### PR TITLE
Turkish translation

### PR DESCRIPTION
**Pull request by serkan-maker**

### CODE REVIEW

File: src/locales/tr.json [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+    "shift": "Shift",
+    "enter": "Enter",
+    "leftClick": "Sol tık",
+    "rightClick": "Sağ tık",
+    "arrowKeys": "Ok tuşları"
+  },
+  "help": {
+    "title": "Yardım",
+    "search": {
+      "text": "Press {ctrl} + {f} to {search}",
```
Comment (D3SOX) [inline src/locales/tr.json:92]: Missing translation?

**Review by D3SOX — COMMENTED** [review]
Thank you for your contribution!

File: src/locales/tr.json [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+    "play": {
+      "text": "{leftClick} um {play}",
+      "play": "oynat"
+    },
+    "options": {
+      "text": "{rightClick} um {moreOptions}",
+      "moreOptions": "daha fazla seçenek göster"
+    },
+    "appPassThrough": {
+      "text": "Press {ctrl} + {g} to {show}",
```
Comment (D3SOX) [inline src/locales/tr.json:104]: Missing `appPassThrough.text`

File: src/locales/tr.json [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      "show": "uygulama geçişini göster"
+    },
+    "reloadSounds": {
+      "text": "Press {ctrl} + {r} to {reload}",
```
Comment (D3SOX) [inline src/locales/tr.json:108]: Missing `reloadSounds.text`

**Comment by D3SOX** [comment]
image
Hmm this seems to be too long for it to fit. Is it possible for you to make it shorter?

**Comment by D3SOX** [comment]
Looks good. One last thing I noticed:
image
The top two options are sorting it by modified date and your translation only says ascending/descending.

**Comment by serkan-maker** [comment]
was changed
[tr.json.zip](https://github.com/Soundux/soundux-ui/files/6445993/tr.json.zip)

**Comment by D3SOX** [comment]
Can you commit it so that I can simply merge this PR? :smile:

**Pull request merged**